src/modules/drawing/components/DrawingToolbox.vue

```diff
@@ -87,6 +108,41 @@ function onDeleteLastPoint() {
                 class="card text-center drawing-toolbox-content shadow-lg rounded-bottom rounded-top-0 rounded-start-0"
                 :class="{ 'rounded-bottom-0': isPhoneMode }"
             >
+                <div
+                    class="d-flex justify-content-center align-items-center gap-2 p-4"
+                    v-if="activeKmlLayer"
```

Comment:
If you could find a way to not hide it at the opening of the drawing module, I would prefer that.
It is a bit weird that the UI "grows" after the first save occurs.

Reply:
I could disable it when there is no active kml layer yet is that ok ?